mozilla-release/browser/components/preferences/sitePermissions.js

```diff
@@ -231,6 +235,9 @@ var gSitePermissionsManager = {
       return;
     let l10nId = this._getCapabilityString(perm.capability);
     let p = new Permission(perm.principal, perm.type, perm.capability, l10nId);
+    // CLIQZ: Skip permissions for Cliqz extension pages
+    if (p.origin.includes(EXTENSION_ID))
```

Comment:
@tsl143 couldn't we check for the type of extension and hide if it's a system addon?

Reply:
We dont get addon/id/type in `perm` object, this maps only URL on which permission is available.
So we need to get URL information, more over, IMO we should move away from allowing special privileges for system addon but spcific cliqz addon, as we install directly https-everywhere from AMO now.

Reply:
cool, makes sense. Thanks!